woocommerce/facebook-for-woocommerce
# Use PSR-4 composer autoloader rather than requiring all classes upfront

### Changes proposed in this Pull Request:

Autoload classes that are already following the PSR-4 naming convention with composer rather than requiring every class upfront.

I removed `require_once` statements for 20 classes, listed below.

Why? It's more efficient, it's quicker for devs and it's less error-prone since the loading happens automatically.

### How to test the changes in this Pull Request:

1. Run `composer install`
2. Check plugin works
3. Run the following code to confirm the 20 classes still load ok:
```php
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Locale::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\AJAX::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Handlers\Connection::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Integrations\Integrations::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Product_Categories::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Products::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Commerce::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Products\Feed::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Products\FBCategories::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Products\Stock::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Products\Sync::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Products\Sync\Background::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\ProductSets\Sync::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Debug\ProfilingLogger::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Debug\ProfilingLoggerProcess::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Utilities\Shipment::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Utilities\Tracker::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Events\Event::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Events\Normalizer::class ) );
var_dump( class_exists( \SkyVerge\WooCommerce\Facebook\Events\AAMSettings::class ) );
```

You should get true for every result if it works.

### Todo

- [x] Confirm this doesn't break Sake (required sake config change)

### Changelog entry

> Dev / New - Use Composer autoloader rather than loading all classes upfront
